### PR TITLE
Fix handling of -0 in budget summary

### PR DESCRIPTION
**Pull request by j-f1**
Before:

CleanShot 2022-08-31 at 13 45 25@2x

### CODE REVIEW

File: packages/loot-design/src/components/budget/rollover/BudgetSummary.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ function TotalsList({ prevMonthName, collapsed }) {
           binding={rolloverBudget.forNextMonth}
           formatter={value => {
             let n = parseInt(value);
-            n = isNaN(n) ? 0 : -n;
+            n = isNaN(n) || n === 0 ? 0 : -n;
             let v = format(n, 'financial');
 
             return n > 0 ? '+' + v : n === 0 ? '-' + v : v;
```
Comment (TomAFrench) [inline packages/loot-design/src/components/budget/rollover/BudgetSummary.js:119]: I'd change this function to be something along the lines of the below which feels more readable to me.

```
let formatter = (value) => {
  let n = parseInt(value) || 0;
  let v = format(Math.abs(n), 'financial');
  return n >= 0 ? '-' + v : '+' + v;
}
```

Comment (j-f1) [reply packages/loot-design/src/components/budget/rollover/BudgetSummary.js:119]: Neat, and `NaN` is indeed falsy.

**Comment by TomAFrench** [comment]
@j-f1 Looks like you edited the wrong function here in 556dc92

**Comment by j-f1** [comment]
Oops, fixed!

**Pull request merged**